src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -39,18 +39,20 @@ public static void initialize(IntakeIO io) {
 
     public Command setAngle(MutableMeasure<Angle> angle) {
         inputs.angleSetPoint = angle;
-        return runOnce(() -> io.setAngle(angle));
+        return runOnce(() -> io.setAngle(Units.Degrees.of( Math.IEEEremainder(angle.in(Units.Degrees), 180)).mutableCopy()));
     }
 
     public Command setAngle(IntakePose intakePose) {
         return setAngle(intakePose.intakePose);
     }
 
     public Command setRollerSpeed(double speed) {
+        inputs.rollerSpeedSetPoint = Units.RotationsPerSecond.of(speed).mutableCopy();
         return runOnce(() -> io.setRollerSpeed(speed));
     }
 
     public Command setCenterRollerSpeed(double speed) {
+        inputs.centerRollerSpeedSetPoint = Units.RotationsPerSecond.of(speed).mutableCopy();
         return runOnce(() -> io.setCenterRollerSpeed(speed));
     }
```

Comment:
We've talked about this before, you need to change the setpoint every time the command is called, not when these functions are called. These functions are called only during compilation.

Reply:
OK